.github/workflows/check-weekly-flaky.yaml

```
@@ -21,6 +21,8 @@ jobs:
     - name: Install Deps
       run: |
         python3 -m pip install -r requirements.txt
+        # Install graphviz for testing graph visualization
+        sudo apt-get install -y graphviz
```

```suggestion
        # Install graphviz for testing graph visualization
        sudo apt-get update
        sudo apt-get install -y graphviz
        python3 -m pip install -r requirements.txt
```
I'm not sure how pip handle the binding but would prefer first installing the system graphviz

---

Ah ok!